### PR TITLE
Add dfn for plural of tick

### PR DESCRIPTION
**Pull request by vkatsikaros**
Fix respec warning.

Found linkless <a> element with text 'ticks' but no matching <dfn>.   respec-w3c-common:1:26900

---
This change is [Reviewable](https://reviewable.io/reviews/w3c/webdriver/475)

### CODE REVIEW

File: webdriver-spec.html [hunk 1]
```diff
@@ -6030,7 +6030,7 @@
         </tr>
       </table>
 
-    <p>A <dfn>tick</dfn> is the basic unit of time over which actions
+    <p>A <dfn data-lt="tick|ticks">tick</dfn> is the basic unit of time over which actions
```
Comment (andreastt) [inline webdriver-spec.html]: The first `tick` is implicit here and can be dropped.

Comment (vkatsikaros) [inline webdriver-spec.html]: Ah I see, I used as example `<dt><dfn data-lt="remote end|remote ends|remote">Remote End</dfn>` https://github.com/w3c/webdriver/blob/master/webdriver-spec.html#L541 

@andreastt in that case is the `remote end` in `data-lt` is due to upper/lower case?

Comment (andreastt) [inline webdriver-spec.html]: It doesn’t care about upper- or lower case, but “remote end” in that list was probably added there before ReSpec supported taking the inner text implicitly. It’s safe to remove it, but that should probably be done in a separate commit.
